### ISSUE TITLE
Include path not properly set to run tests on osx

### ISSUE THREAD

**Issue by olemis** [skycoin/hardware-wallet#91]
**Describe the bug**
`make test` fails due `gcc` not finding project header files.

**Environment information:**

- OS: Mac OS X
- Platform: `Darwin 17.4.0 Darwin Kernel Version 17.4.0: Sun Dec 17 09:19:54 PST 2017; root:xnu-4570.41.2~1/RELEASE_X86_64 x86_64`
- Firmware Version: git checkout at 720fe1ca

**Steps to Reproduce**

1. Change dir to `skycoin/hardware-wallet` folder
2. Run `make test CFLAGS=-I$(brew --prefix sdl2)/include/SDL2/`

**Actual behavior**

```
$ EMULATOR=1 make -C tiny-firmware/emulator/
cc -I/usr/local/opt/sdl2/include/SDL2/ -MMD -MP -o buttons.o -c buttons.c
buttons.c:20:10: fatal error: 'buttons.h' file not found
#include "buttons.h"
         ^~~~~~~~~~~
1 error generated.
make[2]: *** [buttons.o] Error 1
make[1]: *** [emulator] Error 2
make: *** [test] Error 2
```

**Expected behavior**
At that point emulator of Makefile execution, it should had built the `emulator` and continue with tests for `tiny_firmware`

**Additional context**
Since `sdl2` is installed through `homebrew` , if `CFLAGS` is not provided then this is the output .

```sh

$ EMULATOR=1 make test
make -C skycoin-api/ test
./test_skycoin_crypto
Running suite(s): skycoin_crypto
100%: Checks: 14, Failures: 0, Errors: 0
test_skycoin_crypto.c:73:P:checksums:test_generate_public_key_from_seckey:0: Passed
test_skycoin_crypto.c:86:P:checksums:test_generate_key_pair_from_seed:0: Passed
test_skycoin_crypto.c:105:P:checksums:test_secp256k1Hash:0: Passed
test_skycoin_crypto.c:142:P:checksums:test_generate_deterministic_key_pair_iterator:0: Passed
test_skycoin_crypto.c:161:P:checksums:test_base58_address_from_pubkey:0: Passed
test_skycoin_crypto.c:181:P:checksums:test_bitcoin_address_from_pubkey:0: Passed
test_skycoin_crypto.c:201:P:checksums:test_bitcoin_private_address_from_seckey:0: Passed
test_skycoin_crypto.c:224:P:checksums:test_compute_sha256sum:0: Passed
test_skycoin_crypto.c:268:P:checksums:test_compute_ecdh:0: Passed
test_skycoin_crypto.c:362:P:checksums:test_recover_pubkey_from_signed_message:0: Passed
test_skycoin_crypto.c:57:P:checksums:test_base58_decode:0: Passed
test_skycoin_crypto.c:410:P:checksums:test_signature:0: Passed
test_skycoin_crypto.c:420:P:checksums:test_checkdigest:0: Passed
test_skycoin_crypto.c:826:P:checksums:test_addtransactioninput:0: Passed
PASSED ALL TESTS
make emulator
make -C tiny-firmware/vendor/nanopb/generator/proto/
make[2]: Nothing to be done for `all'.
make -C tiny-firmware/protob/
make[2]: Nothing to be done for `all'.
EMULATOR=1 make -C tiny-firmware/emulator/
cc -DEMULATOR=1 -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast -include stdio.h -include /Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/emulator/emulator.h -DHEADLESS=0 -I/usr/include/SDL2 -D_REENTRANT -DCONFIDENTIAL= -O3 -g3 -ggdb3 -std=gnu99 -W -Wall -Wextra -Wimplicit-function-declaration -Wredundant-decls -Wstrict-prototypes -Wundef -Wshadow -Wpointer-arith -Wformat -Wreturn-type -Wsign-compare -Wmultichar -Wformat-nonliteral -Winit-self -Wuninitialized -Wformat-security -Werror -fno-common -fno-exceptions -fvisibility=internal -ffunction-sections -fdata-sections -fstack-protector-all   -DSTM32F2 -DRAND_PLATFORM_INDEPENDENT=1 -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/vendor/libopencm3/include -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/ -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/gen -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/vendor/skycoin-crypto -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/vendor/skycoin-crypto/tools -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/vendor/skycoin-crypto/trezor-tools -I/Users/olemis/Downloads/_/projects/_git/skycoin/hardware-wallet/tiny-firmware/vendor/skycoin-crypto/trezor-tools/aes -DFASTFLASH=0 -DREVERSE_SCREEN=0 -DREVERSE_BUTTONS=0 -DEMULATOR=1 -DDEBUG_RNG=0 -MMD -MP -o buttons.o -c buttons.c
buttons.c:23:10: fatal error: 'SDL.h' file not found
#include <SDL.h>
         ^~~~~~~
1 error generated.
make[2]: *** [buttons.o] Error 1
make[1]: *** [emulator] Error 2
make: *** [test] Error 2
```

**Comment by olemis** [comment]
Reproduced on GNU/Linux as well .

**Comment by olemis** [comment]
fixed by #93 in e96ea83 .
